Apps/WebClient/src/ClientApp/src/models/clinic.ts

```diff
@@ -0,0 +1,23 @@
+// Clinic model
+export default interface Clinic {
+    // Clinic id.
+    clinicId?: string;
```

Comment:
is this going to be undefined at any point? My guess is that if the clinic is undefined this will be fine